packages/webex/src/calling.js

```diff
@@ -4,6 +4,7 @@ import EventEmitter from 'events';
 /* eslint-disable require-jsdoc */
 require('@webex/internal-plugin-device');
 require('@webex/internal-plugin-mercury');
+require('@webex/internal-plugin-encryption');
```

Comment:
What did this change fix?

Reply:
this is needed to make encryption service work in contacts